app/assets/stylesheets/pages/_howitworks.scss

```diff
@@ -1,3 +1,41 @@
+
+
+
+
+.header {
+    margin-top: 5%;
+    padding-left: 0 !important;
+}
+
+.item:hover {
+    color: gray;
+}
+
+body {
```

Comment:
dont add styling to the whole body, also height 100% is default, no need to change